### DISCUSSION TITLE
Per Driver Data

### DISCUSSION THREAD

**Discussion by MrPaulAR** [teslamate-org/teslamate#1784] [Ideas]
Does tesla provide detail as to the active driver?  I realize that this can be changed mid drive but if it is available I would like to be able to link a driver to statistics.

If the driver profile has changed during a drip then the driver that is active for the longest time during a trip or last driver would suffice.

**Comment by cwanja** [comment]
I do not think the driver profile is a currently available API endpoint.

**Comment by JakobLichterfeld** [comment]
Unfortunately not supported by the API: https://github.com/teslamate-org/teslamate/discussions/963#discussioncomment-165631
